<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductWidget</class>
 <widget class="QWidget" name="ProductWidget">
  <property name="windowTitle">
   <string>Продукты</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>Название:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelCategory">
       <property name="text">
        <string>Категория:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboCategory"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelCalories">
       <property name="text">
        <string>Калорийность (на 100 г):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="editCalories">
       <property name="maximum">
        <double>2000.00</double>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkIsRequired">
       <property name="text">
        <string>Обязательный продукт</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnAddProduct">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableProducts"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

